clusterloader2/testing/load/config.yaml

```diff
@@ -194,13 +195,15 @@ steps:
       mediumServicesPerNamespace: {{DivideInt (AddInt $mediumDeploymentsPerNamespace 1) 2}}
       bigServicesPerNamespace: {{DivideInt (AddInt $bigDeploymentsPerNamespace 1) 2}}
 
+{{if $DISABLE_DAEMONSETS}}
```

Comment:
Shouldn't it be {{if not $DISABLE_DAEMONSETS}}??

Reply:
yea, my bad. Done.